### ISSUE TITLE
to json?

### ISSUE THREAD

**Issue by Jossnaz** [BenSampo/laravel-enum#163]
hi, love the project!

I ran into this:

I want to have a 
```php
class Message{
    public MessageType $message_type;
    public string $message;
}
```

With:

```php
namespace App\Enums;
use BenSampo\Enum\Enum;

/**
 * @method static static Success()
 * @method static static Error()
 */
final class MessageType extends Enum
{
    const Success =   'success';
    const Error =   'error';
}

```

when casting to json I want to get

```js
{message: 'adfsadsf adfs asdf ', message_type: 'success'}
```
what I get instead is

```json
[{"message_type":{"value":"success","key":"Success","description":"Success"},"message":"Successfully added matrix entries"}]
```

any shortcut or something?

**Comment by atymic** [comment]
Looks to be a duplicate of #162

**Comment by BenSampo** [comment]
@Jossnaz This should be fixed in v2.2.0. Can you upgrade and try again? Let me know if you're still having issues.
